helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -236,6 +239,9 @@ protected synchronized boolean verifyState() {
       idealStates.entrySet()
           .removeIf(pair -> pair.getValue().getStateModelDefRef().equals(TaskConstants.STATE_MODEL_NAME));
 
+      if (_isLogMore) {
+        System.out.println("---- before verify instances !");
+      }
```

Comment:
Can we get rid of all these println statement in prod code?

Reply:
This is not production code, they are only in verifier which is test utility I believe. The usage is to nail down how long does it take to wait so that we can tune the timeout of test. In github, some operations are fast (say starting Zookeeper)   while some are slow (like refresh cache).

Reply:
removed.